src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -808,25 +789,11 @@ namespace CondenserLoopTowers {
             tower.SizFac = NumArray(33);             //  N21  \field Sizing Factor
             if (tower.SizFac <= 0.0) tower.SizFac = 1.0;
 
-            if (UtilityRoutines::SameString(AlphArray(7), "ScheduledRate")) {
-                tower.BlowdownMode = Blowdown::Schedule;
-            } else if (UtilityRoutines::SameString(AlphArray(7), "ConcentrationRatio")) {
-                tower.BlowdownMode = Blowdown::Concentration;
-            } else if (state.dataIPShortCut->lAlphaFieldBlanks(7)) {
-                tower.BlowdownMode = Blowdown::Concentration;
-                if ((NumNums < 29) && (tower.ConcentrationRatio == 0.0)) {
-                    // assume concentration ratio was omitted and should be defaulted
-                    tower.ConcentrationRatio = 3.0;
-                }
-            } else {
-                ShowSevereError(state, cCurrentModuleObject + '=' + AlphArray(1));
-                ShowContinueError(state, "Invalid " + state.dataIPShortCut->cAlphaFieldNames(7) + '=' + AlphArray(7));
-                ErrorsFound = true;
-            }
```

Comment:
This code section is not needed. GetEnumerationValue() is used instead. The input field also has default value.